src/RestClient/endpoints/V3/Themes/ThemeActions.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeAction } from './types';
+
+class ThemeActions {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Downloads a stores Theme.
+   *
+   * @param uuid A valid store Theme UUID
+   * @returns Promise resolving to a single theme
+   */
+  get(uuid: string): ThemeAction['GetResponse'] {
+    return this.client.post(`${themesPath}/${uuid}/actions/download`);
```

Comment:
I see where your thought process was in naming this `get`, though I'd almost be more inclined to think that `download` might be a better name for this method. The goal of this client is to minimize the context switching that a developer spends between their IDE and our API documentation, and more explicit method names might help with that.